test/image/mocks/zz-bar-rounded-corners.json

```
@@ -0,0 +1,84 @@
+{
+    "data": [
+      {
+        "x": [
+            "giraffes",
+            "orangutans",
+            "monkeys",
+            "capybaras"
+          ],
+          "y": [20, -14, 2, 18],
+          "type": "bar",
+          "marker": {"cornerradius": 10},
+          "name": "Both axes reversed"
+      },
+      {
+        "y": [
+            "giraffes",
+            "orangutans",
+            "monkeys",
+            "capybaras"
+          ],
+          "x": [20, -4, 1, 18],
+          "base": [1, 2, 1, 2],
+          "type": "bar",
+          "orientation": "h",
+        "xaxis": "x2",
+        "yaxis": "y2",
+        "name": "Both axes normal"
+      },
+      {
+        "x": [
+            "giraffes",
+            "orangutans",
+            "monkeys",
+            "capybaras"
+          ],
+          "y": [20, -14, 2, 18],
+          "type": "bar",
+          "marker": {"cornerradius": 10},
```

Let's change one of these to a string to test handling of numbers without percentage.
```suggestion
          "marker": {"cornerradius": "10"},
```

---

@archmoj Visually, this case should display as though the user did not provide a cornerradius, right?

---

(or, should it be treated as an integer?)

---

Actually, I think I just answered my own question -- since `isNumeric` returns `true` for numeric strings, we would treat this value as an integer. (Just double-checking that that seems like the correct behavior @alexcjohnson ?)

---

> @archmoj Visually, this case should display as though the user did not provide a cornerradius, right?

No. We should cast to a number when the string contain a number.